yarn.lock

```diff
@@ -1239,13 +1239,20 @@
     tslint-microsoft-contrib "6.1.0"
     tslint-react "3.6.0"
 
-"@gooddata/typings@^2.23.0", "@gooddata/typings@^2.26.1", "@gooddata/typings@^2.27.0":
+"@gooddata/typings@^2.26.1", "@gooddata/typings@^2.27.0":
   version "2.27.0"
   resolved "https://registry.yarnpkg.com/@gooddata/typings/-/typings-2.27.0.tgz#361888304d56d124cabd9625e81a3b0dae0597f8"
   integrity sha512-vB9/BQHtWHZtTl/LgO/Hsl8xr8YlmHy61kbj7A316Ana08H2wnqV8Y8ZjlVf94bHq9Hza7ViF7urhbCbfP8PFA==
   dependencies:
     lodash "^4.17.11"
 
+"@gooddata/typings@^2.28.0":
```

Comment:
Please deduplicate the typings pacakge

Reply:
Fix pushed (it deduplicated also babel/runtime package)